snorkel/labeling/model/label_model.py

```diff
@@ -58,6 +60,7 @@ class TrainConfig(Config):
     prec_init: float = 0.7
     seed: int = np.random.randint(1e6)
     log_freq: int = 10
+    mu_clamp_min: float = 0.0001
```

Comment:
do we expect anyone to touch this? if no, why 1e-4 and not e.g. 1e-8?

Reply:
Orthogonal questions... pass for now :)

Reply:
Actually... @henryre thoughts on setting it to `None`, and then defaulting to picking something prop. to `1/n`?  Basically that's setting it to allow params to still be sensitive to LFs that only label a few of the `n` data points.

Reply:
See what I just committed